web/src/app/lib/model/attachment.ts

```diff
@@ -15,6 +15,7 @@ export interface Attachment {
 */
 export class Attachment {
     public type?: string;
+    public priceNQT?: string;
```

Comment:
TIL Attachments can have a priceNQT property on them @umbrellacorp03 
Seems this is the case for alias transfers and cancellations